internal/configs/ingress.go

```diff
@@ -181,8 +181,14 @@ func generateNginxCfg(ingEx *IngressEx, pems map[string]string, apResources map[
 
 			ssl := sslServices[path.Backend.ServiceName] || staticParams.SpiffeCerts
 			proxySSLName := generateProxySSLName(path.Backend.ServiceName, ingEx.Ingress.Namespace)
+			var pathType extensions.PathType
```

Comment:
since extensions/v1beta1 is depreciated, shall we start using `networking.k8s.io/v1beta1` (k8s.io/api/networking/v1beta1) in the code as well?

Reply:
I'm doing that part